### ISSUE TITLE
G6 - mobile view - next item after opening drop down isnt the menu

### ISSUE THREAD

**Issue by jszasz** [ccswbs/hjckrrh#125]
On mobile view, if you use keyboard controls (tab), after opening the drop down menu, and using 'tab' to go to the next time, you aren't inside the menu, but select the UoG logo. After pressing the 'tab' button twice more, you finally select the first item in the list 'GryphMail'.

**Comment by brharp** [comment]
I'm not convinced this is a bug. The logo is technically part of the navbar (it is the navbar "brand" in Bootstrap speak). We have followed the suggested order or components for the bootstrap navbar, which puts the toggle button before the brand.

Although there may be a preference for the items to be in a different order, is the current order really an accessibility failure?

**Comment by mmafe** [comment]
It wouldn't constitute a failure, but it is considered an accessibility technique under Focus Order.

http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/SCR26.html

**Comment by jpinsent** [comment]
This looks like it's been resolved. The tab order now has the University of Guelph logo appearing before the menu (there are now more '::before' and '::after' ccs selectors in the code indicating this).

**Comment by mmafe** [comment]
Reopening this since the issue was not committed to develop + master yet. It'll close automatically as soon as the associated pull request (#291) is closed.
